spec/fixtures/float_tasks_api_response.json

```diff
@@ -0,0 +1,78 @@
+{
+  "start_doy": "242",
+  "start_yr": "2015",
+  "people": [
+  {
+    "people_id": "123456",
+    "tasks": [
+      {
+      "end_date": "2015-08-31",
```

Comment:
Hmmm, probably because I did it with the python json.tool on the command line (old habits...yadda, yadda).  I'll see if I can make it neater.